woocommerce/wc-admin
# Add CSS autoprefixer

Fixes #223.

This PR adds `autoprefixer` and `postcss-loader` to our build process, which uses our supported list of browsers (imported from Gutenberg/Core) to generate necessary prefixes.

It does not enable the optional `grid` mode right now for IE11. We might want to use this in the future, but right now it actually makes things worse. See https://github.com/woocommerce/wc-admin/issues/243 for some details about `grid`.

To Test:
* `npm install`
* Look in the generated `dist/index.css` file for `.woocommerce-legend li button .woocommerce-legend__item-container`. Note the `user-select` prefix. 
* You can also run `npx autoprefixer --info` to see what other prefixes are translated based on our browser support. You can try adding another one of these rules to see it translated.
